common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -1444,6 +1444,9 @@
     // imap folder pagination enabled
     public static final KnownKey zimbra_imap_folder_pagination_enabled =  KnownKey.newKey(false);
 
+    // For disabling the printing of jtidy warnings in zmmailboxd.out
+    public static final KnownKey hide_jtidy_warnings = KnownKey.newKey(true);
```

Comment:
Please move to DebugConfig, it is more for debugging.

Reply:
Initially I tried that one only, but it is not getting effective until I explicitly set the parameter.

Reply:
why?  did you debug?

Reply:
Why is it not working same as `jtidy_parser` ? which is also boolean?

Reply:
Done